pkg/operator/k8sutil/network.go

```diff
@@ -104,6 +109,11 @@ func ApplyMultus(net rookv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 	return nil
 }
 
+// getClusterNwApps returns
```

Comment:
```suggestion
// getClusterNetworkApps returns
```

The description is incomplete.

Reply:
Done